.travis.yml

```diff
@@ -0,0 +1,33 @@
+dist: bionic
+
+language: go
+
+go:
+  - 1.12.x
+  - 1.13.x
+
+env:
+  global:
+    - GO111MODULE=on
+
+install:
+  - travis_retry make dependencies
+
+script:
+  - make test
+
+stages:
+ - style
+
+matrix:
+  fast_finish: true
+  include:
+    - stage: style
```

Comment:
Let's ignore the stages and check the style before running the tests. It will be faster.